locator/guice/src/main/java/org/eclipse/kapua/locator/guice/GuiceLocatorImpl.java

```diff
@@ -129,230 +133,64 @@ private void init(String locatorConfigName) throws Exception {
+                classProviderBinder.addBinding()
+                        .toInstance(
+                                new XmlSerializableClassesProvider(
+                                        configurationPrinter,
+                                        locatorConfig.getIncludedPackageNames(),
+                                        locatorConfig.getExcludedPackageNames()));
+                bind(JAXBContextProvider.class).to(JAXBContextProviderImpl.class);
             }
-            configurationPrinter.closeSection();
-        } else {
-            configurationPrinter.addParameter("Excluded XmlSerializable Classes", excludedXmlSerializable.size());
-        }
-
-        // Print it!
-        configurationPrinter.printLog();
-    }
-
-    /**
-     * Adds the given {@link ConfigurationPrinter} the configuration of included and excluded packages.
-     *
-     * @param configurationPrinter The {@link ConfigurationPrinter} to add the configuration to.
-     * @param locatorConfig        The {@link LocatorConfig}.
-     * @since 2.0.0
-     */
-    private void addIncludedExcludedPackageConfig(@NotNull ConfigurationPrinter configurationPrinter, @NotNull LocatorConfig locatorConfig) {
-        // Inclusions
-        configurationPrinter.openSection("Included packages");
-        if (!locatorConfig.getIncludedPackageNames().isEmpty()) {
-            for (String includedPackages : sortedComparable(locatorConfig.getIncludedPackageNames())) {
-                configurationPrinter.addSimpleParameter(includedPackages);
-            }
-        } else {
-            configurationPrinter.addSimpleParameter("None");
-        }
-        configurationPrinter.closeSection();
-
-        // Exclusions
-        configurationPrinter.openSection("Excluded packages");
-        if (!locatorConfig.getExcludedPackageNames().isEmpty()) {
-            for (String excludedPackages : sortedComparable(locatorConfig.getExcludedPackageNames())) {
-                configurationPrinter.addSimpleParameter(excludedPackages);
-            }
-        } else {
-            configurationPrinter.addSimpleParameter("None");
-        }
-        configurationPrinter.closeSection();
-    }
-
-    /**
-     * Sorts the given {@link Collection} of {@link Comparable}s according to the natural order.
-     *
-     * @param strings The {@link Collection} to sort.
-     * @param <C>     The type of {@link Comparable}.
-     * @return The sorted {@link Collection}.
-     * @since 2.0.0
-     */
-    private <C extends Comparable<?>> Collection<C> sortedComparable(Collection<C> strings) {
-        return strings.stream().sorted().collect(Collectors.toList());
+        });
+        //
+        // Create injector
+        injector = Guice.createInjector(instantiatedKapuaModules);
+        XmlUtil.setContextProvider(injector.getInstance(JAXBContextProvider.class));
+        ServiceModuleConfiguration.setConfigurationProvider(() -> injector.getInstance(ServiceModuleProvider.class));
```

Comment:
XmlUtil is initialized here - FOR NOW. As soon as we can drop that static initialization and use proper injection, this line shall have to go